app/uk/gov/hmrc/disareturns/services/InitiateSubmissionDataService.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.disareturns.services
+
+import uk.gov.hmrc.disareturns.models.initiate.inboundRequest.InitiateSubmission
+import uk.gov.hmrc.disareturns.models.initiate.mongo.SubmissionRequest
+import uk.gov.hmrc.disareturns.repositories.InitiateSubmissionRepository
+
+import javax.inject.Inject
+import scala.concurrent.Future
+
+class InitiateSubmissionDataService @Inject() (repository: InitiateSubmissionRepository) {
```

Comment:
where is the unit test spec?

Reply:
Missed this apologies, we've added it in now.